frontend/pages/admin/OrgSettingsPage/cards/constants.ts

```diff
@@ -2,11 +2,17 @@ import { IConfig } from "interfaces/config";
 
 export const DEFAULT_TRANSPARENCY_URL = "https://fleetdm.com/transparency";
 
+export type DeepPartial<T> = T extends object
+  ? {
+      [P in keyof T]?: DeepPartial<T[P]>;
+    }
+  : T;
```

Comment:
So the 2nd logical branch here represents the terminal types of the tree. This is super cool